backend/app/api/v1/users.py

```
@@ -0,0 +1,75 @@
+
+router = APIRouter()
+user_service = UserService()
+
+
+@router.post("/", response_model=UserRead)
+def create_user(user_in: UserCreate, db: Session = Depends(get_session)):
+    """Create a new user."""
+    return user_service.create(db, obj_in=user_in)
+
+
+@router.get("/", response_model=List[UserRead])
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)